Description: disable noexist namespace
 It doesn't give a warning, but fails to test.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Last-Update: 2018-09-22

---

--- thrift-0.11.0.orig/test/ThriftTest.thrift
+++ thrift-0.11.0/test/ThriftTest.thrift
@@ -41,8 +41,8 @@ namespace netcore ThriftAsync.Test
 
 // Presence of namespaces and sub-namespaces for which there is
 // no generator should compile with warnings only
-namespace noexist ThriftTest
-namespace cpp.noexist ThriftTest
+//namespace noexist ThriftTest
+//namespace cpp.noexist ThriftTest
 
 namespace * thrift.test
 
